modules/rostests/winetests/user32/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 
 remove_definitions(-DWINVER=0x502 -D_WIN32_WINNT=0x502)
-add_definitions(-DWINVER=0x602 -D_WIN32_WINNT=0x602)
+add_definitions(-DWINVER=0x600 -D_WIN32_WINNT=0x600)
```

Comment:
Could this be just removed now?

Reply:
If the original add_definitions (so, with 0x602) gets used, this winetest would be compiled with 0x602 (Win8+) definitions of these:
https://github.com/reactos/reactos/commit/d78a5b6dbf9de09d905def711cb0c3f763fcaed8
that would trigger problems as explained in the PR description.

Reply:
I mean removing these two altogether to make it respect the DLLVER we use for building everything.

Reply:
I’d rather make it respect DLLVER,
We can’t change some of the stuff these newer tests require with the NTSHIM for instance. Better to hold off enabling them until we compile ros as NT6 across the board. That’s my two cents